### ISSUE TITLE
Authentic will not show the same date format as Gray Framed

### ISSUE THREAD

**Issue by Sopor-** [qooob/authentic-theme#667]
When running Authentic theme the date format is not the same as using Gray Framed Theme as i have been using for years.
This is how it looks in the Gray Framed Theme: `Time on system 	Tue Feb 28 21:28:48 2017` and when i change theme to Authentic theme it change to this: `Time on system 	Tuesday, February 28, 2017 9:30 PM` when i'm at the 'Dashboard'.
I'm running Webmin in English but i want to have the date format as we have in Sweden, 24H.
If i run 'date' in a terminal window it will show `Tue Feb 28 21:34:24 CET 2017` and this is what Authentic also should show.

Webmin 1.831
Authentic Theme 18.32
Ubuntu Linux Server 14.04.3
Windows 8.1 and Firefox 51.0.1

**Comment by qooob** [comment]
Hi,

It's very easy to make date loom the way you want. Please go to theme configuration page. There you will be able to set short and long dates formats.

**Comment by Sopor-** [comment]
Yeah, i manage to change some of it. Maybe you could include some URL on the configure page that will show these date codes? This was one i found: http://userguide.icu-project.org/formatparse/datetime but when i use those codes it will not show the one i want it to show :(
Maybe you have a better page i can use?
As default you have `LLLL` and if i remove it, save it i get this date format  	2017-02-28T23:36:52+01:00 but when i open the config page those 4 L is back again so if i want to continue without those 4 L i have to remove them everytime i change some of the other settings and before i press Save. If not, those 4 L is back.

**Comment by qooob** [comment]
Hi,

I will add a not about where to look at settings. Have a look at [MomentJS documentation](https://momentjs.com/) page.

Here is the string that you're looking for:

`ddd MMM DD h:mm:ss YYYY`

**Comment by qooob** [comment]
You're welcome.

**Comment by qooob** [comment]
I'm updating _lang_ files to be `momentjs` link more visible.

**Comment by Sopor-** [comment]
I have to ask you why the date format change from 'Thu Mar 16 23:02 2017' to 'tor mar 16 23:04 2017' when i change language and i use 'ddd MMM DD HH:mm YYYY'? I'm only curious because when i have created my own date format it should use that even if i change language.

**Comment by qooob** [comment]
It happens because of momentjs receives as a parameter the current language.

```JavaScript
moment.locale($curr_lang);
```

This is where _Theme Extensions_ come into play and practically let you do quite a bit.

Add the following, editing in extensions `scripts.js`

```JavaScript
t__wi_p.moment.locale('en');
```
